motionplan/plannerOptions.go

```diff
@@ -1,19 +1,26 @@
 package motionplan
 
 import (
+	"fmt"
 	"math"
 	"runtime"
 
+	"github.com/edaniels/golog"
+	pb "go.viam.com/api/service/motion/v1"
 	"gonum.org/v1/gonum/floats"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
 )
 
 // default values for planning options.
 const (
 	// max linear deviation from straight-line between start and goal, in mm.
 	defaultLinearDeviation = 0.1
 
-	// allowable deviation from slerp between start/goal orientations, unit is the norm of the R3AA between start and goal.
-	defaultOrientationDeviation = 0.05
+	// allowable deviation from slerp between start/goal orientations, unit is the number of degrees of rotation away from the most direct
+	// arc from start orientation to goal orientation.
+	defaultOrientationDeviation = 2.0
```

Comment:
why are we changing defaults?

Reply:
This corresponds to the change in `metrics.go` where we now specify orientation deviation in absolute numbers of degrees.

2 degrees is very close to the previous default, within a few hundredths of a degree.